Sources/Amplitude/Plugins/Vendors/AppUtil.swift

```diff
@@ -28,14 +28,24 @@ import Foundation
         }
 
         override var os_name: String {
-            return device.systemName
+            return device.systemName.lowercased()
         }
 
         override var os_version: String {
             return device.systemVersion
         }
 
         override var platform: String {
+            #if os(tvOS)
+                return "tvOS"
+            #elseif targetEnvironment(macCatalyst)
+                return "macOS"
+            #else
+                return "iOS"
```

Comment:
Do we want to cover `watchOS` here?

Reply:
No this is only for `#os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)`. WatchOS is in the later content.